tools/fonts/gonville2smufl.py

```diff
@@ -304,4 +304,90 @@
 # set name / family and generate target file
 f.fontname="GonvilleSmufl";
 f.familyname="GonvilleSmufl";
-f.generate(argv[2], "opentype");
+f.fullname="GonvilleSmufl";
+f.weight="Normal";
```

Comment:
Should this script be moved into the `gonvillesmufl/` folder? The script is the most important part of the GonvilleSmufl font, so maybe it should be distributed alongside the rendered font files.

Reply:
I copied it on your request (rodrigo's font). I would rather delete it now. The source can be found in github (the link is included in the readme file) Maintaing the same file in two locations is not efficient.

Reply:
I agree that there should be one true location for the gonvillesmufl font generator script. Are you maintaining it on your GitHub account? Do you plan to move it into vexflow someday, or do you prefer to keep it separate?